css/mofo-bootstrap.css

```diff
@@ -4895,3 +4895,354 @@ a.bg-danger:focus, a.bg-danger:hover {
 @media print {
   .hidden-print {
     display: none !important; } }
+
+    .bs-slider{
```

Comment:
As this file a third-party dependency from the base fork we use (global sprint), I'd rather not add any custom CSS here. This should go in the main.css file.